### PR TITLE
Added github action

### PR DESCRIPTION
**Pull request by tomjshore**
Added Github action to release when pushing to master

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: .github/workflows/cd.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+          version: next
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Check interesting categories
+        uses: jenkins-infra/interesting-category-action@v1.0.0
+        id: interesting-categories
+        if: steps.verify-ci-status.outputs.result == 'success'
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (corey2604) [inline .github/workflows/cd.yaml:38]: Does this also need `output_result: true` if it is used in  `steps.verify-ci-status.outputs.result == 'success' && steps.interesting-categories.outputs.interesting == 'true'`?

Comment (tomjshore) [reply .github/workflows/cd.yaml:38]: I have copied the workflow from here [https://github.com/jenkinsci/.github/blob/master/workflow-templates/cd.yaml](https://github.com/jenkinsci/.github/blob/master/workflow-templates/cd.yaml)

like the documentation says ```Create .github/workflows/cd.yaml as a copy of the cd.yaml template.```

**Review by corey2604 — APPROVED** [review]

**Comment by daniel-beck** [comment]
Extra `-` in https://github.com/jenkinsci/uleska-plugin/blob/076e95fd3443274b14b8ccb6cecae3e6654a086c/pom.xml#L16 means your version number is now `1.1--rc18.076e95fd3443`

**Comment by daniel-beck** [comment]
@jglick Any idea why the version is an RC in http://updates.jenkins.io/download/plugins/uleska/ ? I checked incrementals docs and other than the extra `-`, it looks correct.

**Comment by jglick** [comment]
The steps in https://www.jenkins.io/doc/developer/publishing/releasing-cd/#pom-file-modifications were omitted.

**Comment by tomjshore** [comment]
oh thank you @daniel-beck and @jglick I see what I have done wrong now.  I will make a new change to the pom file.

**Pull request merged**